Apex Language/APEX_2/OpportunityManager.cls

```diff
@@ -0,0 +1,23 @@
+public class OpportunityManager 
+{
+    public static void updateManagerToAccountInOpp(List<Opportunity> listOpportunity)
+    {
+       Set<Id> opportunityIdSet = new Set<Id>();
+        for(Opportunity opp : listOpportunity)
+        {
+           if(opp.BillToContact__c != null && opp.Manager__c == null)
+           {
+               opportunityIdSet.add(opp.id);
+           }
+       }
+     if(opportunityIdSet.size() > 0)
+     {
+            List<Opportunity> listOpportunityToUpdate = [SELECT BillToContact__r.AccountId,Manager__c FROM Opportunity WHERE id IN : opportunityIdSet];
```

Comment:
check indent

Reply:
is it required to query `Manager__c` field?

Reply:
no didn't require Manager__c